csrc/device_lower/pass/index.cpp

```diff
@@ -1413,14 +1413,14 @@ void IndexLowering::handleCpAsyncBulkLoad(const LoadStoreOp* ldst) {
   auto mbarrier = GpuLower::current()->ldstMBarrierMap().at(ldst);
   auto mbarrier_index = lower_utils::u32IndexScalarSmemTv(mbarrier);
 
+  // gmem indexing and expect_bytes for mbarrier
```

Comment:
Any particular reason of this change (just code movement?)

Reply:
Yeah, just code movement. I just feel that this is slightly easier to read.